examples/messaging/iterating-multiple-subscriptions/java/Main.java

```diff
@@ -0,0 +1,52 @@
+package example;
+
+import io.nats.client.Connection;
+import io.nats.client.Nats;
+import io.nats.client.Dispatcher;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class Main {
+    public static void main(String[] args) {
+        String natsURL = System.getenv("NATS_URL");
+        if (natsURL == null) {
+            natsURL = "nats://127.0.0.1:4222";
+        }
+
+        // Initialize a connection to the server. The connection is AutoCloseable
+        // on exit.
+        try (Connection nc = Nats.connect(natsURL)) {
+
+            // Create a message dispatcher for handling messages in a
+            // separate thread and then subscribe to multiple target subjects.
+            Dispatcher dispatcher = nc.createDispatcher((msg) -> {
+                System.out.printf("Received %s: %s\n",
+                        msg.getSubject(),
+                        new String(msg.getData(), StandardCharsets.UTF_8));
+            });
+
```

Comment:
maybe add a comment before the subscribes
```
// subscribe directly on the dispatcher by subject
```

Reply:
Added it, rephrased a little bit:
```java
// Subscribe directly on the dispatcher for multiple subjects.
```